db/migrate/20201204164913_add_contacts_nhs_number_unique.rb

```diff
@@ -0,0 +1,11 @@
+class AddContactsNhsNumberUnique < ActiveRecord::Migration[6.0]
+  disable_ddl_transaction!
+
+  def up
+    safety_assured { execute 'create unique index index_contacts_on_lower_nhs_number on contacts using btree (lower(nhs_number))' }
+  end
+
+  def dow
```

Comment:
is this a typo? should be 'down'?

Reply:
Yes, thanks, my bad. Sorry about that. Will fix it.